client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -136,12 +142,17 @@ const CellText = React.memo(
     support: bcd.SupportStatement | undefined;
     browser: bcd.BrowserStatement;
   }) => {
-    const currentSupport = getFirst(support);
+    const currentSupport =
+      (support &&
+        asList(support).find((item) =>
+          isFullySupportedWithoutLimitation(item)
+        )) ??
+      getFirst(support);
```

Comment:
Could this be an `getCurrent()` helper function?

This also makes me wonder whether we don't have to rank the support items as follows:

- Active items (`version_removed === null`)
  - Full support without limitation (i.e. only `version_added`)
  - Full support without altname/prefix (i.e. only `version_added` and `notes`)
  - Full support with altname/prefix
  - Partial support
  - ~~No support~~
- Inactive items (`version_removed !=== null`)